### ISSUE TITLE
Players in 1.11 being able to place blocks on their own locations

### ISSUE THREAD

**Issue by lucaazalim** [MylesIsCool/ViaVersion#702]
Hello,

Firstly I would like to say that I'm not sure if this issue is related or not to ViaVersion, so apologies if it's not.

### What is the output link of /viaversion dump? 

https://gist.github.com/anonymous/50510fc458efa5006d194e5e5fdf1c78

### Are you using any additional software like BungeeCord? If so, what software and version? (Please list your plugins as well)

I'm using Waterfall (a BungeeCord fork) with my own plugins.

### How does this error happen? login? Using an item? 

Basically, players using **1.11** are being able to place blocks on their own locations. I also have a [video](https://www.youtube.com/watch?v=PZlbLTW2FEU&feature=youtu.be) showing the problem.

I've tested the problem in 1.8, 1.9, 1.10 and 1.12 and I couldn't reproduce it.

### Is there an error in the console? Use pastebin.com. Is there a kick message?

No.

**Comment by heykadoo** [comment]
Damn

**Comment by heykadoo** [comment]
Insanu

**Comment by heykadoo** [comment]
brbrbrbr hue

**Comment by MylesIsCool** [comment]
When you boot the server does it state that the Paper patch has been applied?

It's because of a bug in paper but we have a workaround that should apply itself otherwise not much we can do.

**Comment by lucaazalim** [comment]
I think it doesn't show any messages saying that the paper patch has been applied. Here is the complete boot log of my server: https://pastebin.com/LFFJ9jCZ

**Comment by MylesIsCool** [comment]
The dump you provided indicated you were running ViaVersion on the PaperSpigot server, the console logs don't contain anything to do with ViaVersion? (This patch will only work if ViaVersion is on the Paper server)

**Comment by lucaazalim** [comment]
I'm sorry, I've pasted the wrong boot log. I got the correct one and yes, it shows a debug on the end of the boot:

``
[15:18:07] [Server thread/INFO]: [ViaVersion] Enabling PaperSpigot/TacoSpigot patch: Fixes block placement.
``

Shouldn't that be fixing the block placement problem?

**Comment by MylesIsCool** [comment]
What specific issues do you get because the patch isn't that good? It's meant as a makeshift workaround and isn't the best due to client lag.

If you can it might be worth switching from Paper / patching the issue (if you want to patch the issue for your version I can probably find the code that would need fixing as someone mentioned it a while back)

**Comment by MylesIsCool** [comment]
See above
